market/connector/google-geo-api/google-geo-api-demo/src_hd/google/getp/map/mapProcess.mod

```diff
@@ -0,0 +1,50 @@
+[Ivy]
+178C91925968397A 9.2.0 #module
+>Proto >Proto Collection #zClass
+ms0 mapProcess Big #zClass
+ms0 RD #cInfo
+ms0 #process
+ms0 @AnnotationInP-0n ai ai #zField
+ms0 @TextInP .type .type #zField
+ms0 @TextInP .processKind .processKind #zField
+ms0 @TextInP .xml .xml #zField
+ms0 @TextInP .responsibility .responsibility #zField
+ms0 @UdInit f0 '' #zField
+ms0 @UdProcessEnd f1 '' #zField
+ms0 @GridStep f3 '' #zField
+ms0 @PushWFArc f4 '' #zField
+ms0 @PushWFArc f2 '' #zField
+>Proto ms0 ms0 mapProcess #zField
+ms0 f0 guid 178C91925CC41AE8 #txt
+ms0 f0 method start() #txt
+ms0 f0 inParameterDecl '<> param;' #txt
+ms0 f0 outParameterDecl '<> result;' #txt
+ms0 f0 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>start()</name>
+    </language>
+</elementInfo>
+' #txt
+ms0 f0 83 51 26 26 -16 15 #rect
+ms0 f1 339 51 26 26 0 12 #rect
+ms0 f3 actionTable 'out=in;
+' #txt
+ms0 f3 actionCode 'in.apiKey = "";' #txt
```

Comment:
use 'ivy.var' and fail if it's not available >> currently it looks as the HD is not usable for the user that didn't read the README.md ... I think we can make it much easier to use if the HD states an error (e.g. by throwing a BpmError). ... that the API key should not be empty.

- [ ] Fail with a clear error for the end-user ... if the API.key is not defined/empty

Reply:
This point already mentioned in variables.yaml
![image](https://user-images.githubusercontent.com/35135581/116080315-1a006880-a6c3-11eb-938e-e869be747ce5.png)

Reply:
Thanks for adding that meaningful variable documentation. I think however, that the user experience could be better if we show a short 'growl' message, in case that the api key is missing. So, the user gets quick feedback on the screen he is looking at, that some configuration must be added. Or is there a technical blocker that prevents that?
![growlWarningForMissingVar](https://user-images.githubusercontent.com/15085693/116097228-3cd25300-a6aa-11eb-8771-099623581df3.png)

Reply:
I added growl message when the API is empty
![image](https://user-images.githubusercontent.com/35135581/116185560-0cdb8c00-a74c-11eb-8444-74c7f65e0e56.png)